hypha/apply/funds/views.py

```diff
@@ -521,29 +553,8 @@ def form_valid(self, form):
             removed=removed,
         )
 
-        # Check if two internal reviewers have been selected.
-        internal_reviewers_count = form.instance.assigned.with_roles().count()
```

Comment:
@Parbhat If I understand correctly "form.instance" and "self.object", used a few lines down, is the exact same thing here?

Reply:
Yeah, it will be submission object.